packages/client/src/components/styledComponents/Profile/Profile.tsx

```diff
@@ -66,20 +95,46 @@ const Error = styled.span`
   font-weight: 400;
   font-size: 10px;
   line-height: 10px;
-  color: ${variables.red000};
+  color: ${variables.red000}!important;
   user-select: none;
 
   &:hover {
     visible: none;
   }
 `
+
+const WrapButtons = styled.div`
+  display: flex;
+  gap: 20px;
+  width: 100%;
+`
+
+const ProfileButton = styled.button`
+  width: 100%;
+  height: 37px;
+  color: ${variables.white000};
+  background-color: ${variables.blue000};
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 500;
+  border-radius: 8px;
+  border: none;
+  cursor: pointer;
+  transition: 0.3s opacity;
+
+  &:hover {
+    opacity: 0.8;
+  }
```

Comment:
У нас же уже есть такой компонент. Единственное отличие - у этого width 100%, а у компонента из ui width 280px. Думаю стоит унаследоваться от существующего компонента с переопределением width.
```suggestion
const ProfileButton = styled(Button)`
    width: 100%;
```